api/python/provisioner/commands/__init__.py

```diff
@@ -685,6 +711,71 @@ def run(self, targets):  # noqa: C901 FIXME
             ) from update_exc
 
 
+@attr.s(auto_attribs=True)
+class RollbackUpdate(CommandParserFillerMixin):
+    input_type: Type[inputs.NoParams] = inputs.NoParams
+    _run_args_type = RunArgsRollbackUpdate
+
+    def run(self, target_version, targets):
+
+        local_minion = local_minion_id()
+        rollback_path = PillarKey('rollback')
+        rollback_pillar = PillarResolver(local_minion).get([rollback_path])
+        rollback_pillar = rollback_pillar[local_minion][rollback_path]
+        cortx_tgt_version = (
+            target_version
+            if target_version in rollback_pillar.keys()
+            else None
+        )
+
+        if cortx_tgt_version is None:
+            logger.exception(
```

Comment:
that makes sense only inside `catch` block. I guess we need to raise something here instead. If you want to log as well please use logger.error or warning depending on the severity.

Reply:
Addressed